### DISCUSSION TITLE
Isn't basic authentication supported?

### DISCUSSION THREAD

**Discussion by zjcnew** [nginx/kubernetes-ingress#1872] [General]
The application I accessed through curl http://user:pwd@abc.test.cn, it not working? It's terrible

Aha! Link: https://nginx.aha.io/features/IC-176

**Comment by pleshakov** [comment]
Hi @zjcnew  This Ingress Controller doesn't support basic auth.  However, there is a workaround to support it described here https://github.com/nginxinc/kubernetes-ingress/issues/200#issuecomment-412231403

**Comment by zjcnew** [comment]
> Hi @zjcnew This Ingress Controller doesn't support basic auth. However, there is a workaround to support it described here [#200 (comment)](https://github.com/nginxinc/kubernetes-ingress/issues/200#issuecomment-412231403)

Please support this feature in the future

**Comment by tomasohaodha** [comment]
Thanks for your suggestion @zjcnew, we will prioritize among the other items in our backlog.

**Comment by saggiyogesh** [comment]
Any ETA, When basic auth feature going to be available in nginx ingress ?

**Reply by brianehlert** [reply]
We have this PR in the queue for review.
Thank you very much for your patience.

**Reply by saggiyogesh** [reply]
Thanks, found a workaround for the time being, using location snippets & config map.

**Comment by brianehlert** [comment]
This was added with release 2.3
